docs/kubectl_update.md

```diff
@@ -21,6 +21,10 @@ $ kubectl update -f pod.json
 
 // Update a pod based on the JSON passed into stdin.
 $ cat pod.json | kubectl update -f -
+
+// Partially update a pod using strategic merge patch
```

Comment:
Mismatch between description and command; command is updating node. Also, mentioning/linking to strategic merge patch is overkill imo. Users don't need to know the implementation detail, as cool as it is.